examples/mfem/bp1.hpp

```diff
@@ -255,5 +255,11 @@ class CeedMassOperator : public mfem::Operator {
     CeedVectorSetArray(u, CEED_MEM_HOST, CEED_USE_POINTER, x.GetData());
     CeedVectorSetArray(v, CEED_MEM_HOST, CEED_USE_POINTER, y.GetData());
     CeedOperatorApply(oper, u, v, CEED_REQUEST_IMMEDIATE);
+
+    const CeedScalar* array;
+    CeedVectorGetArrayRead(u, CEED_MEM_HOST, &array);
```

Comment:
'u' shouldn't need to be synced as its an input

Reply:
Thanks, I thought about this but forgot to modify it.